packages/strapi-design-system/src/Layout/HeaderLayout.js

```diff
@@ -58,6 +58,10 @@ const StickyBox = styled(Box)`
   width: ${(props) => props.width}px;
 `;
 
+const TitleFlex = styled(Flex)`
+  min-width: 0;
+`;
+
```

Comment:
`Flex` inherits from `Box` and it has a [`minWidth`](https://github.com/strapi/design-system/blob/6c4710e399055cb8831c50d3b4a02bfbcc57aa04/packages/strapi-design-system/src/Box/Box.tsx#L105) prop that should be used instead.

Reply:
done